src/main/java/ua/com/danit/dto/response/CommentResponse.java

```diff
@@ -4,8 +4,10 @@
 
 @Data
 public class CommentResponse {
-  public long id;
+
+  public Long id;
   public String text;
-  public long post;
-  public String author;
+  public Long postId;
```

Comment:
are you sure about this? I bet fields postId and authorUsername won't be mapped properly